src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -333,8 +333,13 @@ private void generateSummarySpreadsheet(
         // Vote count
         csvPrinter.print(thisRoundTally);
 
-        // Vote %
-        BigDecimal votePctDivisor = roundTallies.get(round).activeAndLockedInBallotSum();
+        // Vote % (divisor is 1st round total in STV or 1st round determines threshold)
+        BigDecimal votePctDivisor;
+        if (config.isSingleWinnerEnabled() || config.isFirstRoundDeterminesThresholdEnabled()) {
```

Comment:
```suggestion
        if (config.isSingleWinnerEnabled() && !config.isFirstRoundDeterminesThresholdEnabled()) {
```

single winner without firstRoundDeterminesThreshold is the only variation where we want to recalculate the divisor each round. As written currently, the second example in #851 is still calculated incorrectly